keadm/cmd/keadm/app/cmd/debug/get.go

```diff
@@ -0,0 +1,1017 @@
+			}
+
+			converted, err := runtime.Decode(unstructured.UnstructuredJSONScheme, listData)
+			if err != nil {
+				return err
+			}
+			obj = converted
+		} else {
+			obj = podList.Items[0].DeepCopyObject()
+		}
+		if err := PrintGeneric(printer, obj, out); err != nil {
+			return err
+		}
+	}
+	if len(serviceList.Items) != 0 {
+		if len(serviceList.Items) != 1 {
+			for _, info := range serviceList.Items {
+				o := info.DeepCopyObject()
+				list.Items = append(list.Items, runtime.RawExtension{Object: o})
+			}
+
+			listData, err := json.Marshal(list)
+			if err != nil {
+				return err
+			}
+
+			converted, err := runtime.Decode(unstructured.UnstructuredJSONScheme, listData)
+			if err != nil {
+				return err
+			}
+			obj = converted
+		} else {
+			obj = serviceList.Items[0].DeepCopyObject()
+		}
+		if err := PrintGeneric(printer, obj, out); err != nil {
+			return err
+		}
+	}
+	if len(secretList.Items) != 0 {
+		if len(secretList.Items) != 1 {
+			for _, info := range secretList.Items {
+				o := info.DeepCopyObject()
+				list.Items = append(list.Items, runtime.RawExtension{Object: o})
+			}
+
+			listData, err := json.Marshal(list)
+			if err != nil {
+				return err
+			}
+
+			converted, err := runtime.Decode(unstructured.UnstructuredJSONScheme, listData)
+			if err != nil {
+				return err
+			}
+			obj = converted
+		} else {
+			obj = secretList.Items[0].DeepCopyObject()
+		}
+		if err := PrintGeneric(printer, obj, out); err != nil {
+			return err
+		}
+	}
+	if len(configMapList.Items) != 0 {
+		if len(configMapList.Items) != 1 {
+			for _, info := range configMapList.Items {
+				o := info.DeepCopyObject()
+				list.Items = append(list.Items, runtime.RawExtension{Object: o})
+			}
+
+			listData, err := json.Marshal(list)
+			if err != nil {
+				return err
+			}
+
+			converted, err := runtime.Decode(unstructured.UnstructuredJSONScheme, listData)
+			if err != nil {
+				return err
+			}
+			obj = converted
+		} else {
+			obj = configMapList.Items[0].DeepCopyObject()
+		}
+		if err := PrintGeneric(printer, obj, out); err != nil {
+			return err
+		}
+	}
+	if len(endPointsList.Items) != 0 {
+		if len(endPointsList.Items) != 1 {
+			for _, info := range endPointsList.Items {
+				o := info.DeepCopyObject()
+				list.Items = append(list.Items, runtime.RawExtension{Object: o})
+			}
+
+			listData, err := json.Marshal(list)
+			if err != nil {
+				return err
+			}
+
+			converted, err := runtime.Decode(unstructured.UnstructuredJSONScheme, listData)
+			if err != nil {
+				return err
+			}
+			obj = converted
+		} else {
+			obj = endPointsList.Items[0].DeepCopyObject()
+		}
+		if err := PrintGeneric(printer, obj, out); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// HumanReadablePrint Output data in table form
+func HumanReadablePrint(results []dao.Meta, printer printers.ResourcePrinter, out io.Writer) error {
+	podList, serviceList, secretList, configMapList, endPointsList, err := ParseMetaToAPIList(results)
```

Comment:
The return type can be abstracted through interface, only one is returned, and then judged by type.
For example
```
var t interface{}

func main(){
	t  = &api.ServiceList{}
	//articleId  = &api.PodList{}
	switch t.(type) {
	case *api.PodList:
		fmt.Println("I am PodList")
	case *api.ServiceList:
		fmt.Println("I am ServiceList")
	}
}
```
This output : I am ServiceList